packages/input/src/Input.js

```diff
@@ -37,34 +46,21 @@ function Input(props) {
         onBlur
       }) => (
         <Wrapper
-          isDisabled={props.disabled}
+          isDisabled={disabledProp}
           hasFocus={hasFocus}
           hasHover={hasHover}
           variant={props.variant}
```

Comment:
Should this be the `variant` `const` that was declared above?

Reply:
Good catch 👍